src/Evolution/Systems/GrMhd/ValenciaDivClean/KastaunEtAl.cpp

```diff
@@ -297,15 +323,17 @@ Primitives FunctionOfMu<EnforcePhysicality, ThermodynamicDim>::primitives(
     p_hat = get(equation_of_state_.pressure_from_density_and_energy(
         Scalar<double>(rho_hat), Scalar<double>(epsilon_hat)));
   } else if constexpr (ThermodynamicDim == 3) {
-    ERROR("3d EOS not implemented");
+    p_hat = get(equation_of_state_.pressure_from_density_and_energy(
+        Scalar<double>(rho_hat), Scalar<double>(epsilon_hat),
```

Comment:
If you wanted to fix these, I think you need to swap the parens for braces: `Scalar<double>{...}`

Reply:
@nilsvu said to ignore these earlier I think, which is why I didn't change them, but I can change them if that's the way things are going.

Reply:
Sounds good!